src/main/java/com/everest/parkinglotstrategy/ParkByCarStrategy.java

```diff
@@ -0,0 +1,22 @@
+package com.everest.parkinglotstrategy;
+
+import com.everest.parkinglot.ParkingLot;
+import com.everest.parkinglot.Slot;
+import com.everest.vehicle.Vehicle;
+
+public class ParkByCarStrategy extends ParkingStrategy{
+    int slotForCarStartsAt=3;
+    @Override
+    public String parkVehicle(Vehicle vehicle, ParkingLot parkingLot) {
```

Comment:
This is also bad design. Why do we need separate class for each vehicle. What if I add more vehicles? You will end up with more duplicate classes. Design the problem to extend the solution